npm2deb/helper.py

```diff
@@ -26,6 +27,10 @@ def search_for_repository(module):
     found = False
     result = {}
     my_print("Looking for existing repositories:")
+    content = subprocess.check_output(["rmadison", "-u", "debian", "node-"+module])
```

Comment:
Hi Tushar, 
How about using _getstatusoutput instead of check_output, so that you don't have to import subprocess again.